examples/kubernetes/kernel-check/kernel-check.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: bpf-kernel-check
+  name: bpf-kernel-check
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: bpf-kernel-check
+  template:
+    metadata:
+      labels:
+        app: bpf-kernel-check
+    spec:
+      nodeSelector:
+      # run deployment on specific node, e. g. nodename:cilium
+      containers:
+        - name: cilium
+          image: docker.io/cilium/cilium:stable
```

Comment:
I think that we should probably tie this into the current version from the `VERSION` file in the base of the tree, which for the version we merge here can specify `latest` then we'll just need to figure out some extra script to update this version in the tree for when this commit becomes part of a backport branch, eg v1.8 / v1.9.

For now it should suffice to just choose `latest` here. I have a commit in mind that should help us with the second part but it's not merged yet so I can keep that one in mind to fix this up afterwards.

Reply:
done: specified `latest`